frontend/src/components/QuickActionPanel.vue

```diff
@@ -328,6 +352,12 @@ export default defineComponent({
       state.showModal = true;
     };
 
+    const syncDatabaseSchema = () => {
```

Comment:
The feature should only be available for the Enterprise plan. However, the icon should always show up, and we should direct users to get the Enterprise plan to use this feature.

Can be done as a follow-up PR.